project_template/src/operations.c

```diff
@@ -0,0 +1,6 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+#include "operations.h"
+int32_t az_exampleshortname_add_two_numbers(int32_t a, int32_t b) {

```

Comment:
>int32_t [](start = 0, length = 7)

Must include <stdint.h>.
Each source file should handle its own includes otherwise there are hidden dependecies that can be easily broken.